apps/condo/domains/property/components/panels/Builder/BuildingPanelCommon.tsx

```diff
@@ -101,7 +101,7 @@ export const EmptyBuildingBlock: React.FC<IEmptyBuildingBlock> = ({ mode = 'view
             : ''
         const prefix = `${mode}.${generatorAppOrigin ? 'auto' : 'manual'}`
 
-        return intl.formatMessage({ id: `pages.condo.property.EmptyBuildingBlock.${prefix}.EmptyBuildingDescription` }, { services })
+        return intl.formatMessage({ id: `pages.condo.property.EmptyBuildingBlock.${mode === 'view' ? prefix : mode}.EmptyBuildingDescription` }, { services })
```

Comment:
it's better just use two intl messages and use `mode === 'view' ? OneMessage : AnotherMessage`

Reply:
Because it's hard to understand is the translation used or not. Try to avoid dynamic translation keys

Reply:
Done